src/API/Google/Proxy.php

```diff
@@ -137,6 +138,7 @@ public function create_merchant_account(): int {
 	 *
 	 * @throws Exception   When an Exception is caught or we receive an invalid response.
 	 * @throws InvalidTerm When the account name contains invalid terms.
+	 * @throws InvalidDomainName When an Exception related to an invalid top level domain is received.
```

Comment:
Would it be clearer if we just stated "When the site URL contains an invalid top level domain"?

Reply:
I agree. I changed this to "When the site URL ends with an invalid top-level domain".